charts/knative-serving-core/crds/autoscaling-metrics.yaml

```diff
@@ -19,7 +19,11 @@ kind: CustomResourceDefinition
 metadata:
   name: metrics.autoscaling.internal.knative.dev
   labels:
-    serving.knative.dev/release: "v1.0.1"
+    helm.sh/chart: knative-serving-core
```

Comment:
Can we move these repetitive labels to a helper template? And use variables for chart names and versions instead of hardcoding them? 

eg: https://github.com/caraml-dev/helm-charts/blob/main/charts/mlp/templates/_helpers.tpl#L50

Reply:
I wish I could do that, but manifest in `crds` folder can't contain template as of now (https://github.com/helm/helm/issues/7083)

Reply:
Got it.